Dockerfile

```
@@ -0,0 +1,22 @@
+# Use an official Node.js runtime as a parent image
+FROM node:18
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy package.json and package-lock.json to the container
+COPY package*.json ./
+
+# Install project dependencies
+RUN npm install
+
+# Copy the rest of the application code to the container
+COPY . .
```

Combine this with the multi-stage change above.
```suggestion
FROM node:18-slim

COPY --from=build /app/node_modules/ node_modules/
COPY . .
```